assets/js/blocks/classic-template/constants.ts

```diff
@@ -11,7 +11,8 @@ import { TemplateDetails } from './types';
 export const BLOCK_SLUG = 'woocommerce/legacy-template';
 export const TYPES = {
 	singleProduct: 'single-product',
-	archiveProduct: 'archive-product',
+	productCatalog: 'product-catalog',
+	productsBy: 'products-by',
```

Comment:
To me, it's a bit unclear what `productsBy` means. Can we consider other alternative here? Somethings like `productTaxonomy`, `productLoop`, `productGrid`? or even keep `archiveProduct` for shared template, and use `productCatalog` just for the catalog page.